PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.hdinsight/src/com/microsoft/azuretools/hdinsight/projects/SparkLibraryOptionsPanel.java

```diff
@@ -23,62 +23,152 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
-
 import org.eclipse.jface.wizard.WizardPage;
 import org.eclipse.swt.SWT;
 import org.eclipse.swt.events.SelectionAdapter;
 import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.events.SelectionListener;
 import org.eclipse.swt.layout.GridData;
 import org.eclipse.swt.layout.GridLayout;
 import org.eclipse.swt.widgets.Button;
 import org.eclipse.swt.widgets.Combo;
 import org.eclipse.swt.widgets.Composite;
 import org.eclipse.swt.widgets.FileDialog;
-import org.eclipse.swt.widgets.Label;
 import org.eclipse.swt.widgets.Link;
 import org.eclipse.ui.PlatformUI;
 
 import com.microsoft.azure.hdinsight.common.CommonConst;
+import com.microsoft.azure.hdinsight.projects.SparkVersion;
 import com.microsoft.tooling.msservices.components.DefaultLoader;
 import com.microsoft.azuretools.telemetry.AppInsightsClient;
 import com.microsoft.azuretools.hdinsight.Activator;
 import com.microsoft.azuretools.core.utils.Messages;
+import com.microsoft.azuretools.core.utils.PluginUtil;
 
 public class SparkLibraryOptionsPanel extends Composite {
-	private Combo comboBox;
-	private Button button;
+	public boolean useMaven = true;
```

Comment:
>public boolean useMaven = true; [](start = 1, length = 31)

getter/setter?

Reply:
Fixed

---
In reply to: [136773071](https://github.com/Microsoft/azure-tools-for-java/pull/984#discussion_r136773071) [](ancestors = 136773071)